### ISSUE TITLE
Issue putting to nt/NTNDArray PVs

### ISSUE THREAD

**Issue by ddamiani** [mdavidsaver/p4p#24]
I encountered an issue attempting to put to a nt/NTNDArray PV. Below is some example code which causes the issue.

```
import numpy as np
from p4p.client.thread import Context

ctxt = Context('pva')
ctxt.put('demo:pv:name', np.arange(100).reshape(10,10))
```
It gives the following exception
> Exception in Put builder
> Traceback (most recent call last):
>   File "/u1/ddamiani/miniconda3/envs/amii/lib/python3.6/site-packages/p4p/client/raw.py", line 108, in builder
>     V.value = value
> RuntimeError: Only 1-d array can be assigned

**Comment by mdavidsaver** [comment]
This is partially addressed by 7b764dc1d6aba7d1b95f1f182e038b751aef9e0f (in 3.0.1).  The remainder will be addressed through #22.

As a workaround until then, a client can pass the ndarray to be written through `NTNDArray().wrap()`.  eg.

```py
import numpy
from p4p.client.thread import Context
from p4p.nt import NTNDArray
ctxt=Context('pva')
ctxt.put('demo:pv:name', NTNDArray().wrap(numpy.asarray([[1,2],[3,4]])))
```

**Comment by mdavidsaver** [comment]
In P4P==3.1.0
